src/expand.rs

```diff
@@ -220,7 +220,8 @@ impl ExpandedTest {
         let output = normalize_expansion(&output_bytes);
 
         if !expanded.exists() {
-            std::fs::write(expanded, &output)?;
+            // Write a .expanded.rs file contents with an empty line at the end
```

Comment:
FYI it's not an empty line. Text files are defined as having lines that end in a newline character. A file with an empty line at the end would end in two newline characters, one at the end of the empty line and one at the end of the last nonempty line.

Reply:
See https://unix.stackexchange.com/a/18789/179081

Reply:
🤔